services/shhext/envelopes.go

```diff
@@ -0,0 +1,168 @@
+package shhext
+
+import (
+	"sync"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/status-im/status-go/services/shhext/mailservers"
+	whisper "github.com/status-im/whisper/whisperv6"
+)
+
+// EnvelopesMonitor responsible for monitoring whisper envelopes state.
+type EnvelopesMonitor struct {
+	w                      *whisper.Whisper
+	handler                EnvelopeEventsHandler
+	mailServerConfirmation bool
```

Comment:
Does it mean that `EnvelopesMonitor` will send confirmation or what does this flag indicate?

Reply:
this flag means that we expect confirmations from mail server